src/StatusQ/Components/StatusChatListItem.qml

```diff
@@ -95,8 +95,8 @@ Rectangle {
                 if (statusChatListItem.muted && !sensor.containsMouse && !statusChatListItem.highlighted) {
                     return 0.4
                 }
-                return statusChatListItem.hasMention ||
-                        statusChatListItem.hasUnreadMessages ||
+                return statusChatListItem.hasUnreadMessages ||
+                        statusChatListItem.notificationsCount > 0 ||
                         statusChatListItem.selected ||
                         statusChatListItem.highlighted ||
                         statusBadge.visible ||
```

Comment:
Nit: Can remove `statusBadge.visible` from here, since we relying on `notificationsCount` already.

Reply:
@PascalPrecht I am thinking now, since your previous request was that you want to have `statusBadge` exposed to the outside world from this component, then its visibility and value should not be determined inside the component at all. In that case a full and correct flexibility will be achieved by removing lines 21, 99, 142, 150, 189 and 190 from this file and set those 2 props from the outside of this component via exposed `statusBadge`. Any other case where we have `notificationsCount` and `statusBadge` as exposed parameters is kinda mixing contexts cause `notificationsCount` influences to the `statusBadge` value and visibility.